smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/AbstractMultiUserChatIntegrationTest.java

```diff
@@ -136,4 +136,12 @@ static void createModeratedMuc(MultiUserChat muc, Resourcepart resourceName) thr
         answerForm.setAnswer("muc#roomconfig_moderatedroom", true); //TODO Add this to the MucConfigFormManager?
         muc.sendConfigurationForm(answerForm);
     }
+
+    static void createHiddenMuc(MultiUserChat muc, Resourcepart resourceName) throws SmackException.NoResponseException, XMPPException.XMPPErrorException, InterruptedException, MultiUserChatException.MucAlreadyJoinedException, SmackException.NotConnectedException, MultiUserChatException.MissingMucCreationAcknowledgeException, MultiUserChatException.NotAMucServiceException, XmppStringprepException {
+        muc.create(resourceName);
+        Form configForm = muc.getConfigurationForm();
+        FillableForm answerForm = configForm.getFillableForm();
+        answerForm.setAnswer("muc#roomconfig_publicroom", false); //TODO Add this to the MucConfigFormManager?
```

Comment:
nit: space after `//`. I also prefer above the line comments over inline comments. And: Yes, please! :)